src/pages/admin/quizzes/QuizList/index.js

```diff
@@ -92,15 +100,11 @@ const QuizList = () => {
           <td id={style.classColumn}>{quiz.title}</td>
           <td id={style.buttonColumn}>
             <Link to={`/admin/quizzes/${quiz.id}`}>
-              <Button className={style.designButton}>
-                <FaRegEdit size="20px" />
-              </Button>
+              <FaRegEdit size="20px" color="black"/>
             </Link>
           </td>
           <td id={style.buttonColumn}>
-            <Button className={style.designButton}>
-              <RiDeleteBin2Fill size="30px" color="#db7771" />
-            </Button>
+            <RiDeleteBin2Fill size="30px" color="#db7771" />
```

Comment:
the edit and column buttons are present in almost all the tables. is is possible that we can move this to the DataTable component?